### PR TITLE
PICARD-1034: Add support for TOAL id3 frame

### PR DESCRIPTION
**Pull request by samj1912**
https://tickets.metabrainz.org/browse/PICARD-1034

### CODE REVIEW

File: picard/formats/id3.py [hunk 1]
```diff
@@ -112,6 +112,7 @@ class ID3File(File):
         'TCON': 'genre',
         'TMED': 'media',
         'TMOO': 'mood',
+        'TOAL': 'work',
```
Comment (Sophist-UK) [inline picard/formats/id3.py]: Work is also used in freetext.

When saving, the work metadata will be saved both as TOAL and TXXX:work.

When loading and TOAL and TXXX:work are both set, I have not tested which takes precedence or whether you get multi-value metadata.

**Comment by ijabz** [comment]
Please see  discussion at https://community.metabrainz.org/t/picard-using-toal-for-work-in-id3/239131/10

In ID3 TOAL is described as:

The 'Original album/movie/show title' frame is intended for the title
of the original recording (or source of sound), if for example the
music in the file should be a cover of a previously released song.

whereas TIT1 is defined as:

The 'Content group description' frame is used if the sound belongs to
a larger category of sounds/music. For example, classical music is
often sorted in different musical sections (e.g. "Piano Concerto",
"Weather - Hurricane").

So why write works to TOAL it makes no sense to me ?

Also seeing as you are already writing to TXXX:work why are you considering duplicating the information

Also ID3 is case sensitive I thought Picard wrote to TXXX:work not TXXX:Work. 

FWIW Jaikoz/SongKong uses TXXX:WORK rather than TXXX:Work simply because I find that tools such as DbPoweramp Transcoder and MinimServer UPnP expect the same value to be used for different formats otherwise things gets broken.  Also although we use TXXX:WORK we only use it for Classical works rather than any work as that works better for our users, we have a separate field MUSICBRAINZ_WORK_COMPOSITION for any recording that is linked to a MusicBrainz work

Minor point I use TXXX:WORK rather than TXXX:Work simply because I find that tools such as DbPoweramp Transcoder and MinimServer UPnP expect the same value to be used for different formats otherwise things gets broken. But I don't suppose you can/would want to change it at this point.

**Comment by phw** [comment]
We had this change already in the past and reverted again. @Sophist-UK, you did that, maybe you can explain about the reasoning for both introducing and reverting this.

Otherwise I agree with ijabz

**Comment by Sophist-UK** [comment]
I can find where I introduced support for Work - but nothing about using TOAL as the tag for Work.

Can you give me a link to the PR or commit?

**Comment by Sophist-UK** [comment]
P.S. My vote is to use whatever tag has greatest compatibility with other taggers and media players. If they use TOAL for Work, I can go with that - but if they use it as per spec, then I would go with that instead.

**Comment by samj1912** [comment]
No longer relevant after #667

**Comment by Sophist-UK** [comment]
Just as well given the PR no. It was doomed from the start really. :-)

**Pull request closed without merging**